examples/trace/src/main/java/com/google/cloud/opentelemetry/example/trace/TraceExample.java

```diff
@@ -0,0 +1,64 @@
+package com.google.cloud.opentelemetry.example.trace;
+
+import com.google.cloud.opentelemetry.trace.TraceConfiguration;
+import com.google.cloud.opentelemetry.trace.TraceExporter;
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Tracer;
+
+import java.io.IOException;
+import java.time.Duration;
+
+public class TraceExample {
+  private TraceExporter traceExporter;
+
+  private Tracer tracer = OpenTelemetry.getTracer("io.opentelemetry.example.TraceExample");
+
+  private void setupTraceExporter() {
+    // using default project ID
+    TraceConfiguration configuration =
+        TraceConfiguration.builder().setDeadline(Duration.ofMillis(30000)).build();
+
+    try {
+      this.traceExporter = TraceExporter.createWithConfiguration(configuration);
+
+      OpenTelemetrySdk.getTracerProvider()
+          .addSpanProcessor(SimpleSpanProcessor.newBuilder(this.traceExporter).build());
+    } catch (IOException e) {
+      System.out.println("Uncaught Excetption");
+    }
+  }
+
+  private void myUseCase() {
+    // generate a span
+    Span span = this.tracer.spanBuilder("Start my use case").startSpan();
+    span.addEvent("Event 0");
+    // simulate work
+    doWork();
+    span.addEvent("Event 1");
+    span.end();
+  }
+
+  private void doWork() {
```

Comment:
May I ask what's the goal of doWork()?

Reply:
`doWork()` will sleep for a random amount of time to simulate some sort of work, so not every span being exported has the same length, and in addition the span exists for a nonzero amount of time.